lib/action_dispatch/session/dynamo_db_store.rb

```diff
@@ -50,7 +50,8 @@ def delete_session(req, sid, options)
       end
 
       def config_file
-        file = Rails.root.join("config/dynamo_db_session_store/#{Rails.env}.yml")
+        file = ENV.fetch('DYNAMO_DB_SESSION_CONFIG_FILE', nil)
+        file ||= Rails.root.join("config/dynamo_db_session_store/#{Rails.env}.yml")
```

Comment:
Instead of using ||=, potentially just use as the default on fetch

Reply:
I did this originally but I think this reads a lot cleaner.